scipy/_distributor_init.py

```diff
@@ -8,3 +8,8 @@
 The SciPy standard source distribution will not put code in this file, so you
 can safely replace this file with your own version.
 """
+
+try:
```

Comment:
We say above that we're not going to put code in this file, but then we do.

Reply:
> We say above that we're not going to put code in this file, but then we do.

Good point. I'm improving the description in the docstring.

> Is this import only relevant in a dev.py process

Yes - or in editable/pip installs where we rely on the `scipy-openblas` wheels (not implemented yet.

> What happens when we start to delvewheel/delocate/auditwheel in a wheel build?

Nothing different from what happens now. The `_local` file will not exist then, and the name mangling will still be the same.